shared/app.js

```diff
@@ -73,7 +74,9 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
We have a work around for this that we can continue to bundle as desired with this (not ideal, but we can get it to work for now) while we investigate. This way will at least have both ways work for now until we find a better solution - merged and deployed `1.0.3`, also updated example/05_requirejs to reflect these fixes.